docs/src/groups.md

```diff
@@ -75,14 +75,17 @@ implementing the correct `IteratorSize` (i.e. `Base.HasLength()`, or
 optimized to work only based on the type of the group. In particular when the
 information is derivable from the type, there is no need to extend
 
-```@docs
```

Comment:
the sentence above is now unfinished. It should read
```
[...] there is no need to extend [`isfinite`](@ref).
```

Reply:
Agreed, but that's not part of this PR, is it?

Reply:
you moved the docstring that finished it, didn't you? ;)

Reply:
Oh, I didn't realize that was what it did. Honestly, I think that's a bad idea. I'll change it.